### ISSUE TITLE
[3.4] Adding contenttypes breaks the backend.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#7027]
Formerly, if you added a contenttype to `contenttypes.yml`, the backend would tell you to update the DB. In 3.4 it seems that this is throwing an exception instead: 

![screen shot 2017-09-27 at 12 01 25](https://user-images.githubusercontent.com/1833361/30907684-9d108056-a37b-11e7-89c3-4b6387958c50.png)

**Reproduction:**

Set up a bolt 3.4 install, with default contenttypes..

Edit `contenttypes.yml`, add a contenttype: 

```
dummy:
    name: Dummies
    singular_name: Dummy
    fields:
        title:
            type: text
            class: large
            group: content
        slug:
            type: slug
            uses: title
```

Finally, refresh the backend to be greeted by the exception.

**Comment by bobdenotter** [comment]
Full trace: 

![bolt-origin localhost-bolt ipad](https://user-images.githubusercontent.com/1833361/30907831-14e7b8f6-a37c-11e7-8c57-fb5cfb5808d8.png)

**Comment by bobdenotter** [comment]
Fixed in #7028
